api/dbConfig/seed.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO posts (title,name,body)
+VALUES
+(
```

Comment:
you would no need to do this extra configuration if you had js file using fs sync reading the seed.sql file, but interesting approach